G1ANT.Addon.Core/Triggers/FileTrigger.md

```diff
@@ -1,42 +1,45 @@
-# File Trigger
+# FileTrigger
```

Comment:
revert it because it should be the same as Mail Structure for example

Reply:
done